docs/docs/how-to-guides/webops/security/managing-secrets.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 2
+---
+
+# Managing secrets
+
+To manage your FastStore Secrets, you'll use the WebOps Secrets plugin to encrypt and decrypt the Secrets file. When decoded, the secrets file is stored as `secrets.revealed.json` and anyone can see and understand the decrypted key-value pairs of your Secrets—once encrypted, the secrets file extension changes to `secrets.hidden.json`, and every Secret becomes unpredictable and unique.
+
+After committing your code changes into the repository, VTEX IO WebOps will be able to access the `secrets.hidden.json` file, decrypt and consume your secrets as environment variables so the build process can proceed.
+
+![Hide and reveal secrets](/img/how-to-guides/hide-reveal-secrets.gif)
+
+## Before you start
+
+Before proceeding any further with this guide, make sure you have performed the initial setup needed to manage your store secrets. If you're not sure about this prerequisite, check for the `secrets.revealed.json` file at the root of your FastStore project.
+
+For more information, please refer to [Setting up the secrets file](/how-to-guides/webops/security/setting-up-secrets).
+
+## Step by step
+
+### Step 1 - Editing your store secrets
+
+1. Open your FastStore project in any code editor of your choice.
+2. Open the `secrets.revealed.json` file. Notice that the `account` and its encryption-decryption key alias are already defined in the `secrets.revealed.json` file.
+3. According to your scenario, add, modify or delete secrets in the `secrets.revealed.json` file. Notice that secrets must be defined as key-value items as in the following example:
+   
+    ```json
+    {
+        "account": "account-name",
+        "name of the secret": "value of the secret"
```

Comment:
Secret names should obey the follow javascript pattern: "^[a-zA-Z_][a-zA-Z0-9_]*$" . I think important add this information in this doc.

Reply:
Maybe is better to use: "name_of_the_secret" in this example.